common/djangoapps/self_paced/models.py

```diff
@@ -0,0 +1,8 @@
+from config_models.models import ConfigurationModel
+
+
+class SelfPacedConfiguration(ConfigurationModel):
+    """
+    Configuration for self-paced courses.
+    """
+    pass
```

Comment:
Conveniently, if we want to later make something like the verification window configurable, this would be a good place to add that configuration.